internals/search_fulltext.py

```diff
@@ -168,21 +182,24 @@ def canonicalize_string(s: str) -> str:
   return ' ' + canonicalized + ' '  # Avoids matching partial words.
 
 
-def post_process_phrase(phrase: str, feature_ids: list[int]) -> list[int]:
+def post_process_phrase(
+    phrase: str, feature_ids: list[int],
+    field_name: str|None = None) -> list[int]:
   """Fetch the given features and check if they really have the phrase."""
```

Comment:
Nit: update the comment to incorporate  field_name

Reply:
Done.